### PR TITLE
Feature/inp 161

### PR DESCRIPTION
**Pull request by TheJuze**
https://manzoni.atlassian.net/jira/software/projects/INP/boards/71?selectedIssue=INP-161
* added inherit size and color props to the Text component
* restyled first block of home page. Not touched colors

### CODE REVIEW

File: src/components/ArtCardAuthor/index.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+const ArtCardAuthor: FC<IProps> = ({ id, avatar, name, verified = false }) => {
+  return (
+    <div className={styles.artCardAuthor}>
+      <Link to={routes.profile.link(id || '')} className={styles.avatar}>
+        <img src={avatar} alt="" width={24} height={24} className={styles.avatarImg} />
+        {verified ? (
+          <img src={iconVerifiedMark} alt="verified" className={styles.verified} />
+        ) : (
+          <></>
```
Comment (ValnetineKul) [inline src/components/ArtCardAuthor/index.tsx:23]: )

File: src/components/ArtCardAuthor/index.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  name: string;
+  verified?: boolean;
+}
+
+const ArtCardAuthor: FC<IProps> = ({ id, avatar, name, verified = false }) => {
+  return (
+    <div className={styles.artCardAuthor}>
+      <Link to={routes.profile.link(id || '')} className={styles.avatar}>
+        <img src={avatar} alt="" width={24} height={24} className={styles.avatarImg} />
+        {verified ? (
```
Comment (ValnetineKul) [inline src/components/ArtCardAuthor/index.tsx:20]: verified &&

File: src/pages/Home/Banner/components/RandomCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { FC } from 'react';
+import styles from './styles.module.scss';
+import { Text, ArtCardAuthor, EllipsisText } from 'components';
+import mockData from './mockData';
+
+const RandomCard: FC = () => {
+  const { author } = mockData;
+  return (
+    <div className={styles.randomCard}>
+      <img className={styles.art} src={mockData.img} alt={mockData.name} />
```
Comment (ValnetineKul) [inline src/pages/Home/Banner/components/RandomCard/index.tsx:10]: why mock, no api?

File: src/components/ArtCardAuthor/index.tsx (hunk 1 above)
Comment (TheJuze) [reply src/components/ArtCardAuthor/index.tsx:23]: )

**Pull request merged**